spec/features/user_edits_email_spec.rb

```diff
@@ -18,8 +18,7 @@
       visit edit_user_registration_path(user)
 
       fill_in "user[email]", with: "lamhirh.ablairsec@abhnation.com"
-      fill_in "user[current_password]", with: user.password
-      click_button "Update"
+      first('#account > div:nth-child(4) > div').click_button "Update"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.